_posts/2023-01-14-release-2023.01.14.md

```diff
@@ -0,0 +1,136 @@
+---
+title: "Release Day 2023.01.14: Bug hunting"
+description: "Today is release day!"
+published: 2023-01-14
+layout: post
+thumb: https://raw.githubusercontent.com/bottlesdevs/data/main/releases/2022.11.14/release-day.png
+background: https://raw.githubusercontent.com/bottlesdevs/data/main/backgrounds/2022.11.14.png
+published: true
+comments: true
+is_release_day: true
+is_release_day_color: '#FF5F1F'
+user_nick: Kinsteen
+user_name: Kinsteen
+user_tag: Contributor
+co_user_nick: TheEvilSkeleton
+co_user_name: Hari Rana (TheEvilSkeleton)
+co_user_tag: Developer
+---
+
+Happy new Year! Let's hope for the best for everyone!
+
+And it's release day again!
+
+Last month, there wasn't a blog post for the 2022.12.14 release, so everything will be here. It will be a long one!
+
+Lots of stuff to talk about, so let's start with...
+
+## Release schedule change (again)!
+As the project is getting bigger and bigger, and the developers have less time to focus on Bottles, there was a decision to ditch the monthly release cycle, to a cycle similar to GNOME: updates when they are ready. That will let us have more time to develop new features, while still pushing bug fixes in due time, without having to wait a month.
```

Comment:
That's not how GNOME's cycle works – they do it twice a year.

Reply:
Oops that's what I understood with the talks. To be sure, it's release when we want with a numbering scheme *like* GNOME?

Reply:
What should we say instead?

Reply:
I'd just cut out that part here.